src/main/java/com/oneops/api/resource/Assembly.java

```diff
@@ -258,6 +318,67 @@ public CiResource updateAssembly(String assemblyName, String ownerEmail, String
 		String msg = String.format("Failed to update assembly with name %s due to null response", assemblyName);
 		throw new OneOpsClientAPIException(msg);
 	}
+
+	/**
+	 * Updates assembly for the given @assemblyName
+	 *
+	 * @param assemblyName {mandatory}
+	 * @param ownerEmail a valid email address is {mandatory}
+	 * @param description
+	 * @param tags
+	 * @param marketSite
+	 * @return
+	 * @throws OneOpsClientAPIException
+	 */
+	public CiResource updateAssembly(String assemblyName, String ownerEmail, String description, Map<String, String> tags, String marketSite) throws OneOpsClientAPIException {
```

Comment:
@s0i00bu , Please invoke this method from the overloaded method without marketSite by passing marketSite as null, so that we dont duplicate the same lines of code.

Reply:
sure Aneesh. I have done the changes and submitted new PR please review the changes.